app/src/main/res/layout/content_settings_general.xml

```diff
@@ -29,14 +29,15 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
-    <androidx.appcompat.widget.SwitchCompat
-        android:id="@+id/lightThemeToggle"
-        style="@style/SettingsSwitch"
-        android:text="@string/settingsLightTheme"
-        android:theme="@style/SettingsSwitchTheme"
+    <com.duckduckgo.app.settings.SettingsOptionWithSubtitle
+        android:id="@+id/themeToggle"
```

Comment:
can you rename this to `selectedThemeSetting` to follow convention? It's not really a toggle anymore.